packages/usa-modal/src/index.js

```diff
@@ -28,13 +28,6 @@ let modal;
 
 const isActive = () => document.body.classList.contains(ACTIVE_CLASS);
 const SCROLLBAR_WIDTH = ScrollBarWidth();
-const INITIAL_PADDING = window
-  .getComputedStyle(document.body)
-  .getPropertyValue("padding-right");
-const TEMPORARY_PADDING = `${
-  parseInt(INITIAL_PADDING.replace(/px/, ""), 10) +
-  parseInt(SCROLLBAR_WIDTH.replace(/px/, ""), 10)
-}px`;
```

Comment:
> **Note**
> I moved these items so that the styles would be captured after the modal is set up in line 278.

Reply:
Leaving this at the top as-is or via a function helps with clarity. I'd avoid adding more to `init()` if we can avoid it.